content/ri/using-redisinsight/api.md

```diff
@@ -0,0 +1,178 @@
+    "dbType": "OSS_CLUSTER",
+    "seedNodes": [
+        {
+            "host": "redis-cluster-node-1.acme.com",
+            "port": 6379
+        }
+    ]
+}
+```
+
+##### Sentinel-Monitored Database Parameters
+
+Sentinel-monitored databases are added using `dbType: "OSS_SENTINEL"`.
+
+The following additional parameters are required for standalone databases.
+
+| Parameter                     | Type   | Description                                                          |
+|-------------------------------|--------|----------------------------------------------------------------------|
+| sentinelHost                  | string | The hostname or IP address of one of the sentinel instances          |
+| sentinelPort                  | number | The hostname or IP address of one of the sentinel instances          |
+| sentinelPassword              | string | (optional) The password for the sentinel instances                   |
+| sentinelMaster                | object | Information about the monitored database that is to be added.         |
+| sentinelMaster.serviceName    | string | The name of the database to be added. This is the same name used in the `sentinel monitor` directive.
+| sentinelMaster.authPass       | string | (optional) The password, if any, for the monitored database. This can be different from the password of the sentinel instance itself. This is the same password as provided in the `sentinel auth-pass` directive.        |
+| tls        | object | (optional) [TLS parameters for the database](#tls-parameters)      |
+
+
+**Example**
+
+```json
+{
+    "name": "QA Redis Sentinel DB",
+    "dbType": "OSS_SENTINEL",
+    "sentinelHost": "redis-sentinel.acme.com",
+    "sentinelPort": 26379,
+    "sentinelPassword": "sentinel-pass",
+    "sentinelMaster": {
+        "serviceName": "mymaster",
+        "authPass": "opensesame",
+    }
+}
+```
+
+#### TLS Parameters
+
+TLS parameters can be used to specify how RedisInsight should connect to the Redis database over TLS.
+
+The following parameters can be used:
+
+| Parameter                | Type    | Description                                                   |
+|--------------------------|---------|---------------------------------------------------------------|
+| useTls                   | boolean | Whether to use TLS to connect to the database or not          |
+| clientAuth               | boolean | Whether TLS client authentication is required by the database |
+| clientCertificateKeyPair | object  | (optional) [The details of the client certificate and private key used to connect to the Redis database](#tls-client-certificate-and-key). If client authentication is not required, this has to be provided |
+
+##### TLS Client Certificate and Key
+
+The client certificate and key details can be provided in two forms:
+
+1. If the certificate and key has already been used for a database before, the ID of that certificate can be provided directly.
```

Comment:
@nishantgeorge so how do you get the list of id's?  Shouldn't we document the call to get the certificates as well?

Reply:
This section might also benefit from an example after we have that rest call documented

Reply:
Added a new section documenting the endpoint for adding certificates, which returns the ID required to pass here, and liked to that section here.